README.md

```diff
@@ -23,15 +25,58 @@ Terraform 0.12. Pin module version to ~> 2.0. Submit pull-requests to master bra
 
 Terraform 0.11. Pin module version to ~> 1.0. Submit pull-requests to terraform011 branch.
 
-## Usage
-
-### Put an example usage of the module here
+### Usage for combined policy statements
 
 ```hcl
-module "example" {
-  source = "terraform/registry/path"
+// TODO: make sure these references to module names are correct once pushed to the registry
+module "ou_scp" {
+  source = "trussworks/ou-scp"
+  target = {
+    name = "my_ou"
+    id = aws_organizations_organizational_unit.my_ou.id
+  }
+
+  # true means the policy is in effect
+
+  deny_root_account = true
+  deny_leaving_orgs = true
+  deny_creating_iam_users = true
+  deny_deleting_kms_keys = true
+  deny_deleting_route53_zones = true
+  require_s3_encryption = true
+  deny_deleting_cloudwatch_logs = true
+
+  limit_regions = true
+  # - restrict region-specific operations to us-west-2
+  allowed_regions                  = ["us-west-2"]
+
+  protect_s3_buckets = true
+  # - protect terraform statefile bucket
+  protect_s3_bucket_resources = [
+    "arn:aws:s3:::prod-terraform-state-us-west-2",
+    "arn:aws:s3:::prod-terraform-state-us-west-2/*"
+  ]
+
+  protect_iam_roles = true
+  # - protect OrganizationAccountAccessRole
+  protect_iam_role_resources = [
+    "arn:aws:iam::*:role/OrganizationAccountAccessRole"
+  ]
+}
+```
+
+### Usage for a policy which denies all access
 
-  <variables>
+```hcl
+module "scp_test_scp" {
+  # source needs to be the most recent commit hash
+  source = "trussworks/ou-scp"
+  target = {
+    name = "my_ou"
+    id = aws_organizations_organizational_unit.my_ou.id
+  }
```

Comment:
Same thing as above.

Reply:
Thanks. I could've sworn I fixed this already.